daffodil-core/src/main/scala/org/apache/daffodil/core/grammar/AlignedMixin.scala

```diff
@@ -228,29 +235,161 @@ trait AlignedMixin extends GrammarMixin { self: Term =>
       }
     }.value
 
+  private lazy val separatorPrefixMTAApprox =
+    this.optLexicalParent match {
+      case Some(s: SequenceTermBase) if s.hasSeparator =>
+        import SeparatorPosition.*
+        s.separatorPosition match {
+          case Prefix | Infix => LengthMultipleOf(s.knownEncodingAlignmentInBits)
+          case Postfix => LengthMultipleOf(0)
+        }
+      case _ => LengthMultipleOf(0)
```

Comment:
I think this really wants to be AlignmentApprox(1) (MTA is alignment after all so using length isn't quite right) and then with the new `+` operator mentioned above it will work as expected. This is because 1 will always evenly divide the existing alignment so the existing alignment will be used. Same with the other MTA things.

In fact, I *think* `AligmentApprox(0)` never wants to be used except for the root element, since it's kindof a 1-based thing.